src/[sandbox]/sandbox.mock.ts

```diff
@@ -275,8 +275,9 @@ export const profileMock = [
 ];
 
 export const presetTagsMock: any = {
```

Comment:
@futurethang would you want to rename this mock to match the new property name `contactData`?

Reply:
yeP!